modules/mobile/spec/request/v1/messages_request_spec.rb

```diff
@@ -165,6 +165,18 @@
         expect(response.parsed_body['data'].any? { |m| m['id'] == thread_id.to_s }).to be false
       end
 
+      it 'does not filter the provided message when excludeProvidedMessage is false' do
```

Comment:
Should we also have a test for when it's omitted? Or is it marked as required in the docs

Reply:
Nvm, the base test covers that. LGTM